hazelcast/src/main/java/com/hazelcast/internal/nearcache/NearCache.java

```diff
@@ -58,8 +58,24 @@
             = new HazelcastProperty(PROP_EXPIRATION_TASK_PERIOD_SECONDS,
             DEFAULT_EXPIRATION_TASK_PERIOD_SECONDS);
 
+    /**
+     * Indicates how a near cache is updated. Adding
+     * new entry to near cache or updating an existing
+     * one are all the subject of this setting.
+     */
```

Comment:
Minor: maybe "Indicates how a near cache is updated." is enough? The second sentence is not that clear and IMHO not really necessary.

Reply:
fixed